conf/mesos-site.xml

```diff
@@ -89,6 +89,21 @@
     <value>3</value>
   </property>
 
+  <property>
+    <name>mesos.hdfs.jre-url</name>
+    <value>https://downloads.mesosphere.io/java/jre-7u76-linux-x64.tar.gz</value>
```

Comment:
I didn't want to have empty values for the JRE related config values here (in the non-Mesosphere) config.  Suggestions about appropriate values are welcome.